src/pages/components/button/usage.mdx

```diff
@@ -542,107 +539,805 @@ recommended action labels.
+
+Ghost buttons work well as a cancel button in progressive flows, as they draw
+less attention; users have to purposefully find and click it to cancel. In
+tearsheets, the buttons are fluid which wouldn’t be a suitable application for a
+tertiary button.
+
+<Row>
+<Column colLg={12}>
+
+![Example of ghost button used in a modal.](images/button_usage_ghost_9.png)
+
+</Column>
+</Row>
+
+<Caption>Example of ghost button used in a modal.</Caption>
+
+## Danger button
+
+Danger button is used for actions that could have destructive effects, such as
+“Delete”, “Remove”, or “Stop”.
+
+The danger button has three different styles:
+[primary](/components/button/style#danger-primary-button-color.mdx),
+[tertiary](/components/button/style#danger-tertiary-button-color.mdx), and
+[ghost](/components/button/style#danger-ghost-button-color.mdx). Determining
+which danger button style to use will depend on the level of emphasis you want
+to give to the destructive action. Destructive actions that are a required or
+primary step in a workflow should use the primary danger button style. However,
+if a destructive action is just one of several actions a user could choose from,
+then a lower emphasis style like the tertiary danger button or the ghost danger
+button may be more appropriate.
+
+<Row>
+<Column colLg={8}>
+
+![Danger button in the context of a product UI.](images/button_usage_danger_primary.png)
+
+</Column>
+</Row>
+
+<Caption>Danger button in the context of a product UI.</Caption>
+
+<DoDontRow>
+  <DoDont type="do" caption="Do use a tertiary danger button for lower emphasis destructive actions.">
+
+![Do use a tertiary danger button for lower emphasis destructive actions.](images/button_usage_danger_tertiary.png)
+
+  </DoDont>
+  <DoDont type="do" caption="Do use a ghost danger button for lower emphasis destructive actions.">
+
+![Do use a ghost danger button for lower emphasis destructive actions.](images/button_usage_danger_ghost.png)
+
+  </DoDont>
+</DoDontRow>
 
 ## Modifiers
 
-### Buttons with icons
+### Button with icon
 
-Icons can be placed next to labels to both clarify an action and call attention
-to a button. There are certain instances where an icon will suffice in place of
-a text label, but use icon buttons cautiously.
+Icons can be placed next to labels to clarify an action and call attention to a
+button. However, icons should be used sparingly, as overuse can create visual
+noise and make an experience less usable. If you use a button with an icon in
+one part of your UI it does not mean that you need to add icons to all other
+buttons.
 
 - Use 16px glyphs within buttons; use 20px glyphs within the large expressive
-  buttons
-- Glyphs are distinguished by their solid shape and knocked-out details
-- Glyphs should always appear to the right of the text
+  buttons.
+- Glyphs are distinguished by their solid shape and knocked-out details.
+- Glyphs should always appear to the right of the text.
```

Comment:
Why are we saying "Glyphs" here in all these bullet points, isn't this the rule for icons?